neuromation/api/storage.py

```diff
@@ -451,10 +451,9 @@ def _is_remote_modified(self, local: os.stat_result, remote: FileStatus) -> bool
         except (FileExistsError, neuromation.api.IllegalArgumentError):
             raise NotADirectoryError(errno.ENOTDIR, "Not a directory", str(dst))
         await queue.put((progress.enter, StorageProgressEnterDir(src, dst)))
+        loop = asyncio.get_event_loop()
         async with self._file_sem:
-            folder = sorted(
```

Comment:
I'm curious why sorting is not needed anymore.

Reply:
Because in any case tasks start concurrently. It is not predicable what operation will start or finish first.

Reply:
got it